From 62f035f018c99792b804a9c8e43915e23e8e97ec Mon Sep 17 00:00:00 2001
From: Laurentiu Palcu <laurentiu.palcu@nxp.com>
Date: Mon, 18 Dec 2017 08:22:25 +0200
Subject: [PATCH 3126/5242] MLK-17232-2: drm: imx: dcss: ignore
 SB_PEND_DISP_ACTIVE interrupt

commit  c8e0613e76644f657cf8248ce81c44114c157b5d from
https://source.codeaurora.org/external/imx/linux-imx.git

There is a logic error in the DCSS B0 silicon and this interrupt does
not behave as it's supposed to. Ignore for now.

Signed-off-by: Laurentiu Palcu <laurentiu.palcu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/imx/dcss/dcss-ctxld.c |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/gpu/imx/dcss/dcss-ctxld.c b/drivers/gpu/imx/dcss/dcss-ctxld.c
index c3ce95f..e5076f9 100644
--- a/drivers/gpu/imx/dcss/dcss-ctxld.c
+++ b/drivers/gpu/imx/dcss/dcss-ctxld.c
@@ -225,8 +225,7 @@ void dcss_ctxld_hw_cfg(struct dcss_soc *dcss)
 	struct dcss_ctxld_priv *ctxld = dcss->ctxld_priv;
 
 	dcss_writel(RD_ERR_EN | DB_COMP_EN | SB_HP_COMP_EN | SB_LP_COMP_EN |
-		    DB_PEND_SB_REC_EN | SB_PEND_DISP_ACTIVE_EN | AHB_ERR_EN |
-		    RD_ERR | AHB_ERR,
+		    DB_PEND_SB_REC_EN | AHB_ERR_EN | RD_ERR | AHB_ERR,
 		    ctxld->ctxld_reg + DCSS_CTXLD_CONTROL_STATUS);
 }
 
-- 
1.7.9.5

